test/functorch/test_dispatch.py

```diff
@@ -0,0 +1,115 @@
+# Owner(s): ["module: functorch"]
+from functools import wraps
+import unittest
+
```

Comment:
nit: test_vmap_registrations is a better name for this file.

Also is there a reason why you didn't put all of this into functorch/test_vmap? (I'm neutral either way)

Reply:
> nit: test_vmap_registrations is a better name for this file.

👍 

No, I was going off the the similar `test_find_dangling_impls` which was in test/test_dispatch.py

Reply:
done in https://github.com/pytorch/pytorch/pull/89465/commits/249ace286845b208455bc8b73cde7e84d49bc852